testsV2/common_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package tests contains end to end tests meant to verify the Cloud SQL Auth proxy
+// works as expected when executed as a binary.
+//
+// Required flags:
+//    -mysql_conn_name, -db_user, -db_pass
+package tests
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/cloudsql-proxy/v2/cmd"
+	"github.com/spf13/cobra"
+)
+
+// proxyExec represents an execution of the Cloud SQL proxy.
```

Comment:
There's a mismatch betweeen the doc and the type. I think unexported makes sense.